activestorage/app/models/active_storage/variation.rb

```diff
@@ -59,14 +59,14 @@ def transform(file, &block)
 
   def format
     transformations.fetch(:format, :png).tap do |format|
-      if MiniMime.lookup_by_extension(format.to_s).nil?
+      if Marcel::Magic.by_extension(format.to_s).nil?
```

Comment:
Benchmark: Marcel is faster here- 

https://gist.github.com/vipulnsward/e897aedcc9b239018b1b60db7575be5c